Yatima/ForLurkRepo/FixName.lean

```diff
@@ -22,6 +22,12 @@ def validCharsTree : RBTree Char compare :=
     'q', 'r', 'w', 'x', 'Q', 'R', '-', ':',
     '_']
 
+def charToHex (c : Char) : String :=
+  let n  := Char.toNat c;
+  let d2 := n / 16;
+  let d1 := n % 16;
+  hexDigitRepr d2 ++ hexDigitRepr d1
```

Comment:
Do you know why they've removed this function from core? 👀

Reply:
Seems like it was a "misleading" name? 
https://github.com/leanprover/lean4/commit/8781203372836f99db48154b2b963bcaf169b9e0